DYDX-Bots/Insolvent_Contract/src/findings.ts

```diff
@@ -0,0 +1,17 @@
+import { Finding, FindingType, FindingSeverity } from "forta-agent";
+import { BigNumber } from "ethers";
+
+export const createFinding = (totalBorrowerDebtBalance: BigNumber, totalActiveBalanceCurrentEpoch: BigNumber) => {
+  return Finding.fromObject({
+    name: "Contract Insolvent",
```

Comment:
```suggestion
    name: "Liquidity Module Contract is insolvent",
```
It would be better to include what contract is insolvent.